.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -7,9 +7,11 @@ about: Use this template for raising a feature request
 **Describe the feature and the current behavior/state.**
 
 **Relevant information**
+- Have you read the requirements for new contributions in [`CONTRIBUTING.md`](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#requirements-for-new-contributions-to-the-repository)? (yes/no):
```

Comment:
I suppose something more like:
If you want to contribute read the requirements...

Reply:
I'll include it as a note after the "Are you willing to contribute" bullet point.